### ISSUE TITLE
Vaccinazioni complete

### ISSUE THREAD

**Issue by RiccardoBorchi** [italia/covid19-opendata-vaccini#57]
Consiglio di inserire il numero di vaccinazioni complete (con la seconda dose) in questo file: https://github.com/italia/covid19-opendata-vaccini/blob/master/dati/vaccini-summary-latest.json

**Comment by LucaZeta** [comment]
E' stata implementata la differenziazione prime/seconde dosi rendendo, fino alla somministrazioni di vaccini monodose, che ancora non avviene, completamente controllabile il monitorig della campagna di vaccinazione.

Complimenti per efficacia e tempismo nell'apportare le modifiche necassarie all'open-dataset pubblicato in questo spazio.
Esemplare.

**Comment by CloudItaly** [comment]
Aprendo il file JSON, non sono ancora presenti le voci differenziate.

**Comment by LucaZeta** [comment]
> non sono ancora presenti le voci differenziate

Prime e seconde dosi non sono indicate per ciascuna categoria, ma sono disponibili al livello superiore.

**Comment by RiccardoBorchi** [comment]
> E' stata implementata la differenziazione prime/seconde dosi rendendo, fino alla somministrazioni di vaccini monodose, che ancora non avviene, completamente controllabile il monitorig della campagna di vaccinazione.
> 
> Complimenti per efficacia e tempismo nell'apportare le modifiche necassarie all'open-dataset pubblicato in questo spazio.
> Esemplare.

no, mi riferivo a inserire il totale delle vaccinazioni complete, in questo file: https://github.com/italia/covid19-opendata-vaccini/blob/master/dati/vaccini-summary-latest.json

Il numero delle seconde dosi somministrate giornalmente, per ciascuna Regione, l'ho visto. Io mi riferisco al TOTALE delle vaccinazioni complete.

**Comment by LucaZeta** [comment]
> mi riferivo a inserire il totale delle vaccinazioni complete, in questo file: https://github.com/italia/covid19-opendata-vaccini/blob/master/dati/vaccini-summary-latest.json

Ho letto male il nome del file al quale ti stai riferendo.
Scusa la confusione.

**Comment by RiccardoBorchi** [comment]
> > mi riferivo a inserire il totale delle vaccinazioni complete, in questo file: https://github.com/italia/covid19-opendata-vaccini/blob/master/dati/vaccini-summary-latest.json
> 
> Ho letto male il nome del file al quale ti stai riferendo.
> Scusa la confusione.

Nessun problema!
